Update the version strings to 2.9.1 and add New Features (#69)
SiliconLabsSoftware/docs-matter: sld290-matter-wifi-getting-started-example/index.md

## Setting up the Matter over Wi-Fi Development Environment

> Refer to the [Release Notes](https://github.com/SiliconLabs/matter_extension/releases/tag/v2.9.1) to know more about the latest releases from Silicon Labs.

Reviewer: ```suggestion
For more information about the latest Silicon Labs releases, see the [Release Notes](https://github.com/SiliconLabs/matter_extension/releases/tag/v2.9.1).
```
